tests/js/module-string-names/module-string-names.js

```diff
@@ -0,0 +1,7 @@
+export { smile as "smile" } from "./emojis.js";
+export { "smile" as smile } from "./emojis.js";
+export { "smile" as "smile" } from "./emojis.js";
+export { foo, bar as "foo" } from "./emojis.js";
+export { foo, bar as "foo" };
+export { "學而時習之，不亦說乎？", "吾道一以貫之。" as "忠恕。" } from "Confucius";
+export * as "foo", { default as "quux" } from "module-b";
```

Comment:
We only have tests for export. Because babel-parser can't parse import when used with the flow plugin. I reported to babel repo: https://github.com/babel/babel/issues/12209

Reply:
Let's add one case in `errors`, so we won't forget it.

Reply:
f1182e7 👍